lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss

```diff
@@ -4,7 +4,7 @@
     color: $white;
     margin: auto;
     max-width: 1440px;
-    padding: $base-spacing 0 $section-spacing;
+    padding: $base-spacing 0 12px;
```

Comment:
Made this section-spacing again per Luke's orders, but the fix below took care of the problem anyway so not big deal